core/src/banking_stage.rs

```diff
@@ -443,46 +443,46 @@ impl BankingStage {
         // Many banks that process transactions in parallel.
         let bank_thread_hdls: Vec<JoinHandle<()>> = (0..num_threads)
             .map(|i| {
-                let (verified_receiver, unprocessed_transaction_storage) =
+                let (packet_receiver, unprocessed_transaction_storage) =
```

Comment:
i also considered `batch_receiver` but i opted for `packet_receiver`:

- `batch` is actually implementation details. in the context of BankingStage construction, it should be irrelevant.

also, i also avoided `transaction_receiver`:

- use of the low level term _packet_ is so wide in codebase to fight against it ;)
- at this context, i think readers can figure out there must be some conversion from the received low-level thing called _packets_ to more high level thing called _transactions_ before storing them into `unprocessed_..._storage`, *by not using _transaction_receiver_ here*. Otherwise, it would indicate simple path-through. (in fact, it's not, ref: `ImmutableDeserializedPacket` wrapper)

Reply:
Imo the naming makes sense - it also matches the type `BankingPacketReceiver` given the current context is Banking(Stage)